script/Ciel/Day 4/04_01_CIEL04_1B.txt

```diff
@@ -86,68 +90,72 @@ She was standing near the window, so I guess she was airing the room out, as she
 [sha:34a0924f23266caaf74b5014b5bfe8fa195f0012]{
 -- Page 44, Offset 8282.
 -- 「翡翠。悪いんだけど、替えの上着ってあるかな。いま着ているの、昨日の雨で濡れちゃって」
-"I'm sorry Hisui but do you have a spare jacket? The one I'm wearing got wet in the rain yesterday."
+"Hisui―――sorry, but is there any chance we have a spare jacket somewhere?
```

Comment:
i can't read this without shiki having a british accent. Old version is good. Though, sub "wet" for "soaked"

Reply:
Assuming the 'by any chance' is the offender? Asking if _Hisui_ has a spare jacket (personally) reads slightly off to me.
> "Hisui―――sorry, but do we have any spare jackets?
>  The one I'm wearing is soaked through from the rain yesterday."

less :soubrit:?

Reply:
yep, that's fine